test/framework/options.py

```diff
@@ -198,6 +198,16 @@ def test_force(self):
 
         self.assertTrue(not re.search(already_msg, outtxt), "Already installed message not there with --force")
 
+        # check that --rebuild works
+        args = [
+                eb_file,
+                '--rebuild',
+                '--debug',
+               ]
```

Comment:
this is basically copy-paste from above, how about this to clean this whole block (and the stuff for `--force` above) up a bit:

```python
# check that --force and --rebuild work
for arg in ['--force', '--rebuild']:
    outtxt = self.eb_main([eb_file, '--debug', arg])
    self.assertTrue(not re.search(already_msg, outtxt), "Already installed message not there with %s" % arg)
```